### ISSUE TITLE
`tye deploy -i` when pod is in crash backoff doesn't deploy new version

### ISSUE THREAD

**Issue by glennc** [dotnet/tye#126]
Deployed an app that wasn't reading config properly, saw that it was crashing on startup, change code and run deploy again.

Expected: new version of app running
Actual: Still running previous

**Comment by rynowak** [comment]
yep. So far we've been relying on k8s behavior for this. We probably need to what helm and other tools do (delete pods, or scale down and back up).

**Comment by glennc** [comment]
I like content hashing better I think. But haven't spent lots of time thinking about it.
